test/suites/ai-model-serving/ai-model-serving-online.robot

```diff
@@ -0,0 +1,126 @@
+*** Settings ***
+Documentation       Sanity test for AI Model Serving
+
+Library             OperatingSystem
+Library             String
+Library             Process
+Library             ../../resources/DataFormats.py
+Resource            ../../resources/common.resource
+Resource            ../../resources/oc.resource
+
+Suite Setup         Setup Suite
+Suite Teardown      Teardown Suite
+
+
+*** Variables ***
+${USHIFT_HOST}=             ${EMPTY}
+${MODEL_NAME}=              openvino-resnet
+${DOMAIN}=                  ${MODEL_NAME}-predictor-test-ai.apps.example.com
+${NAMESPACE}=               test-ai
```

Comment:
For online test we could randomize the namespace like it's done in other suites.

Reply:
Good idea, I'll fix it in next commit.